pageserver/src/http/openapi_spec.yml

```diff
@@ -398,7 +398,7 @@ paths:
           content:
             application/json:
               schema:
-                type: string
+                $ref: "#/components/schemas/LsnByTimestampResponse"
```

Comment:
You have to also specify path parameter `version`, see https://swagger.io/docs/specification/describing-parameters/#path-parameters

Otherwise, it won't be possible to pass this parameter via generated clients (in most cases), I suppose

Reply:
I've added the version parameter, good catch!